recipes/editorconfig-core

```diff
@@ -1,3 +1,3 @@
-(editorconfig-core :repo "editorconfig/editorconfig-emacs"
+(editorconfig-core :repo "10sr/editorconfig-core-emacslisp"
```

Comment:
I moved `editorconfig-core` to editorconfig/editorconfig-emacs so no need to change this :innocent: 
`10sr/editorconfig-core-emacslisp` is not maintained any more.